lib/react/server_rendering/separate_server_bundle_container.rb

```diff
@@ -0,0 +1,18 @@
+require 'open-uri'
+
+module React
+  module ServerRendering
+    # Get a compiled file from Webpacker's output path
+    class SeparateServerBundleContainer
+
+      def self.compatible?
+        !!defined?(Webpacker)
```

Comment:
Check for `Shakapacker`?

Reply:
I don't want to try supporting Shakapacker v6 & v7 at the same time.